packages/expo-updates/ios/EXUpdates/EXUpdatesDevLauncherController.m

```diff
@@ -0,0 +1,119 @@
+//  Copyright © 2021 650 Industries. All rights reserved.
+
+#import <EXUpdates/EXUpdatesAppController+Internal.h>
+#import <EXUpdates/EXUpdatesAppLauncherWithDatabase.h>
+#import <EXUpdates/EXUpdatesConfig.h>
+#import <EXUpdates/EXUpdatesDevLauncherController.h>
+#import <EXUpdates/EXUpdatesReaper.h>
+#import <EXUpdates/EXUpdatesReaperSelectionPolicyDevelopmentClient.h>
+#import <EXUpdates/EXUpdatesRemoteAppLoader.h>
+#import <EXUpdates/EXUpdatesSelectionPolicy.h>
+#import <EXUpdates/EXUpdatesUpdate.h>
+#import <React/RCTBridge.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+static NSString * const EXUpdatesDevLauncherControllerErrorDomain = @"EXUpdatesDevLauncherController";
+
+@implementation EXUpdatesDevLauncherController
+
+@synthesize bridge = _bridge;
```

Comment:
```suggestion
@property bridge;
```
I believe this the "auto synthesis" that you get from just declaring a @property and is no longer required.

Reply:
The `@property` is defined on the protocol (EXUpdatesInterface) this class is implementing, which is one of those cases where you need to synthesize the variable manually. I suppose I could redefine the property directly on this class's interface, but I'd prefer not to have two copies of the same property definition if there's no need 🤷‍♂️